### ISSUE TITLE
Separate crafting thresholds

### ISSUE THREAD

**Issue by Quizer85** [PrincessRTFM/AutoKittens#10]
The new feature of automatically crafting non-storage-limited resources based on the availability of their inputs looks useful, but we're going to need to be able to set thresholds separately for each crafting operation. Unfortunately this looks like it will necessitate a complete overhaul of the UI of that section of the script options.

**Comment by VariableVixen** [comment]
Correct. The UI is already getting bloated and should probably be split into separate panels for separate sections, adding in this level of detailed control will absolutely demand it. I agree it's an important feature though.
